rosbag2_cpp/include/rosbag2_cpp/clocks/player_clock.hpp

```diff
@@ -77,6 +77,18 @@ class PlayerClock
   ROSBAG2_CPP_PUBLIC
   virtual bool sleep_until(rclcpp::Time time) = 0;
 
+  /**
+   * \return whether the clock is currently sleeping.
+   */
+  virtual bool is_sleeping() = 0;
```

Comment:
Missed `ROSBAG2_CPP_PUBLIC` ?

Reply:
Fixed: 49a0e9369ff3aa39ed82d1c258a760e7df25d622

and rebased on `rolling`